apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/AnyAssertionSamples.kt

```diff
@@ -201,25 +201,22 @@ class AnyAssertionSamples {
         }
     }
 
+    interface Person {
+        val age: Int
+    }
+
+    private val customers = emptyList<Person>()
+
```

Comment:
I copy this inside of the test so that it is part of KDoc in the end, not ideal but better than nothing I would say

Reply:
I intentionally did not do this, because I think it only distracts. I think the examples are easy to read without seeing the actual types. So I’d prefer not to have it in the samples, but will of course move it if you prefer it that way.

Reply:
ok, then keep it like this